app/controllers/api/tasks_controller.rb

```diff
@@ -9,5 +9,9 @@ def index
     def show
       render_resource stack.tasks.find(params[:id])
     end
+
+    def trigger
+      render_resource stack.trigger_task(params[:task_name], current_user), status: :accepted
+    end
```

Comment:
We might want to update the `require_permission` call for `TasksController` as `trigger` is more then a `:read` on `:stack`.